CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,75 @@
+* Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+
+In this community we strive to go the extra step to look out for each other. Don’t just aim to be technically unimpeachable, try to be your best self. In particular, avoid flirting with offensive or sensitive issues, particularly if they’re off-topic; this all too often leads to unnecessary fights, hurt feelings, and damaged trust; worse, it can drive people away from the community entirely.
+
+And if someone takes issue with something you said or did, resist the urge to be defensive. Just stop doing what it was they complained about and apologize. Even if you feel you were misinterpreted or unfairly accused, chances are good there was something you could’ve communicated better — remember that it’s your responsibility to make your fellow Rustaceans comfortable. Everyone wants to get along and we are all here first and foremost because we want to talk about cool technology. You will find that people will be eager to assume good intent and forgive as long as you earn their trust.
+
+## Harassment
+
+_Our definition of harassment is based on the [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md) and the [WeAllJS Code of Conduct](https://wealljs.org/code-of-conduct)._
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+* Violence, threats of violence or violent language directed against another person.
+* Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+* Posting or displaying sexually explicit or violent material.
+* Posting or threatening to post other people's personally identifying information ("doxing"). This includes any sort of "outing" of any aspect of someone's identity without their consent.
+* Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+* Inappropriate photography or recording.
+* Inappropriate physical contact. You should have someone's consent before touching them.
+* Unwelcome sexual attention, including when simulated online. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
+* The use of sexualized language or imagery. The only exception to sexual topics is channels/spaces specifically for topics of sexual identity.
+* Deliberate intimidation, stalking or following (online or in person).
+* Making light of/making mocking comments about trigger warnings and content warnings.
+* Any of the above behaviour even when presented as "ironic" or "joking".
+* Any attempt to present "reverse-ism" versions of the above as violations. Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia".
```

Comment:
These are the only lines that differ from Rust CoC, and they are taken from WeAllJS CoC. I think these are quite relevant for internet communities and the gaming community in particular, and provides reassurance for new contributors that we don't tolerate this kind of behaviour.